pkgs/by-name/um/umu-launcher/package.nix

```
@@ -0,0 +1,15 @@
+{ buildFHSEnv, umu-launcher-unwrapped }:
+buildFHSEnv {
+  pname = "umu-launcher";
+  inherit (umu-launcher-unwrapped) version meta;
+
+  targetPkgs = pkgs: [ pkgs.umu-launcher-unwrapped ];
+
+  executableName = "umu-run";
```

IDK if this is good practice, but this could be set as:
```suggestion
  executableName = umu-launcher-unwrapped.meta.mainProgram;
```